homeassistant/components/rako/config_flow.py

```diff
@@ -0,0 +1,82 @@
+"""Config flow for Rako."""
+import asyncio
+from typing import Any, Dict, Optional
+
+from python_rako import Bridge, discover_bridge
+from python_rako.const import RAKO_BRIDGE_DEFAULT_PORT
+from python_rako.exceptions import RakoBridgeError
+from python_rako.model import BridgeInfo
+import voluptuous as vol
+
+from homeassistant.config_entries import CONN_CLASS_LOCAL_PUSH, ConfigFlow
+from homeassistant.const import CONF_BASE, CONF_HOST, CONF_PORT
+from homeassistant.data_entry_flow import FlowResultDict
```

Comment:
It was renamed in https://github.com/home-assistant/core/pull/49847, please rebase.